packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -661,6 +664,156 @@ export const NotifiTargetContextProvider: FC<PropsWithChildren> = ({
     [],
   );
 
+  const getSignature = useCallback(
+    async (message: ArrayLike<number> | string) => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const signature = await walletWithSignParams.signMessage(message);
+      let hexString = '0x';
+
+      for (let i = 0; i < Object.keys(signature).length; i++) {
+        // Convert each value to a two-digit hexadecimal string and append it to the result
```

Comment:
Not sure if this `getSignature` is going to be an general method that will adopt different wallet? or only for coinbase.
Because we supports multi-chain, the signature might vary. Not sure if you have an idea how to handle those case?